client/state/themes/selectors.js

```diff
@@ -417,10 +417,9 @@ export function getThemeHelpUrl( state, themeId, siteId ) {
  * @return {?String}         Theme purchase URL
  */
 export function getThemePurchaseUrl( state, themeId, siteId ) {
-	if ( isJetpackSite( state, siteId ) || ! isThemePremium( state, themeId ) ) {
+	if ( ! isThemePremium( state, themeId ) ) {
```

Comment:
oops yeah, makes sense. After moving it to the other selector I should've restored this.